Please ignore error in `NetworkValidation`, it's not related to this PR, it failed due to NuGet not being able to resolve dependencies. I've already reported this error to SDK & Azure PowerShell teams.

```
D:\a\1\s\Out\azure-powershell\src\Network\Network\Network.csproj : error NU1605: Detected package downgrade: Microsoft.Rest.ClientRuntime from 2.3.20 to 2.3.19. Reference the package directly from the project to select a different version.  [D:\a\1\s\Out\azure-powershell\src\Network\Network.sln]
```

--- a/specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/azureFirewall.json
+++ b/specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/azureFirewall.json
@@ -242,8 +242,7 @@
           "Update Azure Firewall Tags": {
             "$ref": "./examples/AzureFirewallUpdateTags.json"
           }
-        },
-        "x-ms-long-running-operation": true
+        }
       }
     },
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/azureFirewalls": {

